tests/conftest.py

```diff
@@ -103,14 +108,69 @@ def init_teams(g):
         ('identifier3', 'name3'),
         ('identifier4', 'name4'),
     ]
+    # Add teams.
+    created_teams = []
+    for team in init_teams:
+        vteam = g.addV('Team') \
+            .property(T.id, str(uuid.uuid4())) \
+            .property(Cardinality.single, 'identifier', team[0]) \
+            .property(Cardinality.single, 'name', team[1]) \
+            .sideEffect(
+                __.addE("universe_of")
+                .from_(
+                    __.V()
+                    .hasLabel('Universe')
+                    .has('namespace', CURRENT_UNIVERSE.namespace)
+                    .has('version', CURRENT_UNIVERSE.version.int_version)
+                )
+            ) \
+            .elementMap() \
+            .next()
+        created_teams.append(DbTeam.from_vteam(vteam))
+
+    created_teams.sort(key=lambda x: x.vid)
 
+    yield created_teams
+
+
+@pytest.fixture
+def old_universe():
+    """Creates a universe with version 0.0.1 so it can be used in tests as a
+    universe older than the current one.``."""
+    yield Universe(
+        UniverseVersion.from_int_version(
+          CURRENT_UNIVERSE.version.int_version + 1
```

Comment:
Shouldn't `int_version` be a method call?

Reply:
No, it's not needed when a property is defined using the``@property`` decorator:
https://docs.python.org/3/library/functions.html?highlight=property#property